### ISSUE TITLE
[4.0] administrator module edit. Dropdown Save&Close fails when module User Status deactivated

### ISSUE THREAD

**Issue by ReLater** [joomla/joomla-cms#24911]
### Steps to reproduce the issue
- Backend
- Go to modules > Administrator modules.
- Deactivate all modules of type "User Status".
- Open module "Title" of Type "Title" (many others have the same problem but type "Custom")
- Click on button `Save & Close` to open dropdown submenu.

### Expected result
- Dropdown opens.
![15-05-_2019_11-29-02](https://user-images.githubusercontent.com/20780646/57765029-f4137080-7704-11e9-9fb8-c080615f19c9.jpg)

### Actual result
- Dropdown doesn't open. Can't select "Save and Copy" etc.

### Additional comments
- I think it's missing JQuery and/or bootstrap(?) that is loaded by module "User Status" with following line:
`HTMLHelper::_('bootstrap.framework');`

**Comment by infograf768** [comment]
Confirmed.
Are missing
```
<script src="/installmulti/joomla40/media/vendor/jquery/js/jquery.min.js?3.4.1"></script>
<script src="/installmulti/joomla40/media/vendor/bootstrap/js/bootstrap.bundle.min.js?4.3.1"></script>
```

**Comment by infograf768** [comment]
Issue can be solved by adding
`HTMLHelper::_('bootstrap.framework');`
in
`/administrator/components/com_modules/tmpl/module/edit.php`

**Comment by ReLater** [comment]
I'll provide a pr.

**Comment by ghost** [comment]
Closed as having Pull Request #24921
